src/js/rx/components/OrderHistory.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import moment from 'moment';
+
+import TrackPackageLink from './TrackPackageLink';
+
+class OrderHistory extends React.Component {
+  makeRow(item, index) {
+    const attrs = item.attributes;
+    return (
+      <tr key={index}>
+        <td>
+          Shipped on {moment(
+            attrs['shipped-date']
+          ).format('MMM D, YYYY')}
+        </td>
+        <td>
+          <TrackPackageLink
+              text={attrs['tracking-number']}/>
+          &nbsp;({attrs['delivery-service']})
+        </td>
+      </tr>
+    );
+  }
+
+  render() {
+    const rows = this.props.items.map(this.makeRow);
+
+    return (
+      <table className={this.props.className}>
+        <thead>
+          <tr>
+            <th>Order status</th>
```

Comment:
There's currently no way to pull order date from the tracking history, so @ayaleloehr said to leave it off for now.